packages/compass-e2e-tests/tests/collection-bulk-delete.test.ts

```diff
@@ -162,15 +162,12 @@ describe('Bulk Delete', () => {
       useBuilders: false,
     });
     expect(text).to.equal(`from pymongo import MongoClient
-
```

Comment:
Because I remove the line extra breaks in the custom command to make it consistent across environments for easier assertions.